src/framework/ui/view/iconcodes.h

```diff
@@ -302,7 +302,7 @@ class IconCode
 
         TOOLBAR_GRIP = 0xF3A2,
 
-        PAGE = 0xF3A5,
+        PAGE = 0xF3AE,    //f3a5
```

Comment:
What's this?

Reply:
Same as Camera???

Reply:
Sorry, I was testing something else. I forgot to change it back